browser/src/control/Control.DocumentNameInput.js

```diff
@@ -23,6 +23,7 @@ L.Control.DocumentNameInput = L.Control.extend({
 	},
 
 	documentNameConfirm: function(value) {
+		$('#document-name-input-loading-bar').css('display', 'block');
```

Comment:
hmm, I think this function only sends message to the server that we want to rename.
If animation has to be finished after renaming was finished - probably it has to be handled somewhere else, maybe here: https://github.com/CollaboraOnline/online/blob/3922137e27fecf7afc40b931bc1bfadc4b10612c/browser/src/core/Socket.js#L1295

Did you test for example with Nextcloud with large file? It should take a bit to save as or rename.

Reply:
I have made changes, and now it's working fine for large files. Thanks for letting me know!